addons/crm/static/src/views/crm_kanban/crm_kanban_view.js

```diff
@@ -9,7 +9,7 @@ import { CrmKanbanRenderer } from "@crm/views/crm_kanban/crm_kanban_renderer";
 export const crmKanbanView = {
     ...kanbanView,
     ArchParser: CrmKanbanArchParser,
-    // Makes it easier to patch
+    // Makes it easier to patch // ???
```

Comment:
`crm_iap_mine` adds some features to the controller by patching it, since we don't have any special features to add in `crm` I though this was an easier way to do it (in case another module not depending on crm_iap_mine would need to patch it aswell)